GOVERNANCE.md

```diff
@@ -0,0 +1,125 @@
+
+#### Lead Maintainer
+The Maintainer team includes one person in a Lead role,
+who is responsible for coordinating the activity of the group.
+In addition to the responsibilities listed for all Maintainers above,
+the Lead Maintainer:
+
+* schedules Maintainer meetings
+* prepares [the agenda for Maintainer meetings](#meeting-agenda)
+* shares the agenda with all Maintainers at least 24 hours before the meeting
+* [assigns roles at the start of each meeting](#meeting-roles)
+* schedules official releases of the lesson
+* acts as a point of contact for the Maintainer team
+* invites other community members to Maintainer meetings as non-voting participants
+
+Where needed e.g. due to absence,
+the Lead Maintainer may defer any of these responsibilities to another member of the Maintainer team.
+
+The Lead Maintainer has a term length of 12 months,
+and it is expected that the role will rotate among members of the Maintainer team.
+If a Maintainer is up next in the rotation and wishes to decline the role of Lead 
+e.g. due to a lack of capacity,
+they should communicate that with the other Maintainers at the earliest opportunity
+to help the team plan accordingly.
+
+#### Current Maintainers
+See [README.md](./README.md) for a list of the current project Maintainers.
+
+#### Joining/Leaving the Maintainer Team
+Maintainers volunteer to take on the role, and other members of the community may 
+volunteer to join the Maintainer team at any time,
+or be invited by the existing Maintainers.
+Additions to the Maintainer team will be discussed and approved by the current membership.
+No formal onboarding exists for new Maintainers,
+but some informal onboarding can be expected from the existing Maintainers.
+
+Maintainers may step away from the role at any time,
+but are expected to communicate the decision to the whole Maintainer team
+and to coordinate with other Maintainers to transfer responsibilities, e.g.
+re-assign issues, resolve outstanding pull requests, etc.
+
+### Contributors
+Anyone who opens or comments on an issue or pull request,
+or who provides feedback on the curriculum through another means,
+is considered to be a Contributor to the project.
+
+Maintainers are responsible for ensuring that all such contributions are credited,
+e.g. on the curriculum site and/or when (and if) a release of the curriculum is made to Zenodo.
+
+Contributors of more singificant changed to the lesson may be invited by the Maintainers to add themselves to the 
+Authors list.
+
+## Maintainer Meetings
+The Maintainer team meets frequently,
+at minimum for at least 30 minutes four times per year.
+Meetings provide an opportunity for Maintainers to
+discuss outstanding issues and pull requests
+and co-work on the project where necessary.
+
+### Meeting schedule
+See [README.md](./README.md) for details of the current Maintainer meeting schedule.
+
+### Meeting agenda
+The agenda for Maintainer meetings will be prepared as a collaborative document,
```

Comment:
Shall we just include the link for the current Google doc here? And then change if/when we move to a different operation model.

Reply:
We could add the link here but I'd again argue not to have it in the GOVERNANCE file which is probably not meant to be updated often. I can add the link to the README but we are also not required to make this document public (even if people will likely not be able to access it). Let me know what you think is preferred - a link in README or no link at all @bielsnohr.

Reply:
OK, I have now actually updated both README and GOVERNANCE docs with details of meetings and meetings notes. It cannot hurt to have them in both locations. Hope this is OK @bielsnohr?

Reply:
Yep, fine by me. As you say, doesn't really hurt to have in both locations. I appreciate the `GOVERNANCE` doc probably shouldn't change that much, but I do feel that meetings are an important part of governance and some reference to them should be made here.